.github/ISSUE_TEMPLATE/implementing-new-platform.md

```diff
@@ -0,0 +1,55 @@
+- [ ] [stream-metadata-go](https://github.com/coreos/stream-metadata-go)
+    - [Example PR](https://github.com/coreos/stream-metadata-go/pull/45/)
+    - [ ] `Add platform to the Media struct in release/release.go`
+    - [ ] `Add supporting code for new platform to _toStreamArch_ func found in the release/translate.go` 
+    - [ ] (Cloud Only) `Cloud Images need to have a Images field`
+- [ ] (Cloud Only) [stream-metadata-rust](https://github.com/coreos/stream-metadata-rust/)
+    - [Example PR](https://github.com/coreos/stream-metadata-rust/pull/16)
+- [ ] [build pipeline](https://github.com/coreos/fedora-coreos-pipeline)
+    - [Example PR](https://github.com/coreos/fedora-coreos-pipeline/pull/500)
+    - [ ] `Add platform to the list found in jobs/build.Jenkinsfile for building the new artifact`
+- [ ] [fedora-coreos-browser](https://github.com/coreos/fedora-coreos-browser)
+    - [Example PR](https://github.com/coreos/fedora-coreos-browser/pull/35)
+    - [ ] `Add a list element for new platform in browser/index.html`
+- [ ] [fedora-coreos-docs](https://github.com/coreos/fedora-coreos-docs)
+    - [Example PR](https://github.com/coreos/fedora-coreos-docs/pull/377)
+    - [ ] `Add a provisioning-[platform].adoc that walks through how to setup the new platform` 
+
+## At Release
+
+1. Merge:
+    - [ ] Platform support enhancements made it to coreos-assembler
+    - [ ] Fedora website changes
+    - [ ] Stream-metadata-go changes
+    - [ ] Fedora-coreos-browser changes
+2. Wait for updates made to coreos-assembler to be propagated to latest container
+    - [ ] Download Latest version of coreos-assembler container. Verify platform support functionality. 
+3. Merge:
+    - [ ] Build pipeline changes
+4. Tag for bot pickup: [see semantic versioning](https://semver.org)
+    - [ ] Long tag stream-metadata-go commit with an appropriate new version.
+    - [ ] Long tag fedora-coreos-stream-generator commit with a new appropriate version.
+    ```
+    #Ensure gpg key for signing in github settings that is associated to redhat email.
+    git status
+    #Verify you are on the upstream repo's main branch.
+    git tag -s v#.#.#
+    #replace '#' with numbers.
+    #give appropriate detail to tag, check pervious tags with 'git show v#.#.#'
+    git push git@github.com:coreos/targeted-repo.git v#.#.#
+    #navigate to the targeted-repo's tag section to ensure a valid signed tag is listed [e.g. stream-metadata-go](https://github.com/coreos/stream-metadata-go/tags) 
+    ```
+5. Wait for Dependabot to create and merge PR's from the new tags.
+    - This can be triggered manually by navigating to [coreos-assembler's Dependabot](https://github.com/coreos/coreos-assembler/network/updates) && [fedora-coreos-stream-generator's Dependabot](https://github.com/coreos/fedora-coreos-stream-generator/network/updates/). This might need to be done a few times, as the Dependabot might not pickup tag changes for a few attempts after initial tagging.
```

Comment:
Just navigating there isn't enough; need to click the link and then "Check for updates".